src/com/lftechnology/java/training/sanish/application/model/Model.java

```diff
@@ -0,0 +1,46 @@
+package com.lftechnology.java.training.sanish.application.model;
+
+import java.sql.ResultSet;
+
+/**
+ * Db model interface
+ *
+ * @param <M> {@link Model}
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public interface Model <M extends Model>{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) M is not used in the interface. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2326)